Rewrite `Contests` (#5784)
ppy/osu-wiki: wiki/Contests/en.md

| [2013](https://osu.ppy.sh/home/news/2014-01-04-best-maps-of-2013-voting) | ![osu!][osu!]/![osu!taiko][osu!taiko]/![osu!catch][osu!catch]/![osu!mania][osu!mania] | [Infected Mushroom - The Pretender](https://osu.ppy.sh/beatmapsets/79498) hosted by ![][flag_US] [RLC](https://osu.ppy.sh/users/1047883) | [Linked Horizon - Guren no Yumiya (TV Size)](https://osu.ppy.sh/beatmapsets/87630) hosted by ![][flag_ID] [Kuria](https://osu.ppy.sh/users/870067) (*osu!taiko difficulties by* ![][flag_TW] *[qoot8123](https://osu.ppy.sh/users/766371))* | [ONE OK ROCK - Rock, Scissors, Paper](https://osu.ppy.sh/beatmapsets/82282) hosted by ![][flag_DE] [Haya](https://osu.ppy.sh/users/870067) (*osu!catch difficulties by* ![][flag_DE] *[Tenshichan](https://osu.ppy.sh/users/1101600))* | [Wotamin - Gigantic O.T.N](https://osu.ppy.sh/beatmapsets/80214) hosted by ![][flag_KR] [Star Stream](https://osu.ppy.sh/users/389236) (*osu!mania difficulties by* ![][flag_PH] *[Entozer](https://osu.ppy.sh/users/277044) and* ![][flag_KR] *[Xellop](https://osu.ppy.sh/users/376604))* | [Results](https://osu.ppy.sh/home/news/2014-02-10-best-maps-of-2013-results) |

> | [2014](https://osu.ppy.sh/home/news/2015-01-11-best-of-2014-vote-now) | ![osu!][osu!]/![osu!taiko][osu!taiko]/![osu!catch][osu!catch]/![osu!mania][osu!mania] | [Aoi Eir - IGNITE (TV size ver.)](https://osu.ppy.sh/beatmapsets/190222) hosted by ![][flag_JP] [Guy](https://osu.ppy.sh/users/91738) | [Hanatan - Airman ga Taosenai](https://osu.ppy.sh/beatmapsets/134151) hosted by ![][flag_SV] [Natsu](https://osu.ppy.sh/users/1953876) (*osu!taiko difficulties by* ![][flag_ID] *[mingmichael](https://osu.ppy.sh/users/1570876) and* ![][flag_HK] *[aabc271](https://osu.ppy.sh/users/155707))* | [Hanatan - Airman ga Taosenai](https://osu.ppy.sh/beatmapsets/134151) hosted by ![][flag_SV] [Natsu](https://osu.ppy.sh/users/1953876) (*osu!catch difficulties by* ![][flag_PE] *[\[Yue\]](https://osu.ppy.sh/users/645486) and* ![][flag_AR] *[Mythiax](https://osu.ppy.sh/users/1834516))* | [Hanatan - Airman ga Taosenai](https://osu.ppy.sh/beatmapsets/134151) hosted by ![][flag_SV] [Natsu](https://osu.ppy.sh/users/1953876) (*osu!mania difficulties by* ![][flag_ID] *[- R u m i a -](https://osu.ppy.sh/users/645486) and* ![][flag_VN] *[Kuo Kyoka](https://osu.ppy.sh/users/2596306))* | [Results](https://osu.ppy.sh/home/news/2015-02-01-best-of-2014-results) |

Reviewer: - R u m i a - is now Rumia-

#### [Pending Cup \#7 (2019)](https://osu.ppy.sh/community/forums/topics/953416)

| Song | Mode | ![Gold Crown][GCrown] | ![Silver Crown][SCrown] | ![Bronze Crown][BCrown] | Results |
| :-- | :-: | :-- | :-- | :-- | :-- |
| kamome sano - archive::zip | ![osu!][osu!] | ![][flag_FR] [Realazy](https://osu.ppy.sh/users/873961) | ![][flag_RU] [Senseabel](https://osu.ppy.sh/users/6184386) | ![][flag_US] [Axarious](https://osu.ppy.sh/users/2614511) | [Results](https://osu.ppy.sh/community/forums/topics/982516) |
| EmoCo. - This Club is not 4 U | ![osu!][osu!] | ![][flag_CN] [Ellyu](https://osu.ppy.sh/users/4438154) | ![][flag_DE] [Lasse](https://osu.ppy.sh/users/896613) | ![][flag_RS] [Seolv](https://osu.ppy.sh/users/8067876) | [Results](https://osu.ppy.sh/community/forums/topics/982516) |
| BEMANI Sound Team "Nekomata Master" - The last of world music | ![osu!][osu!] | ![][flag_PL] [Kalibe](https://osu.ppy.sh/users/3376777) | ![][flag_CA] [J1\_](https://osu.ppy.sh/users/5918561) | ![][flag_KR] [Heilia](https://osu.ppy.sh/users/9823042) | [Results](https://osu.ppy.sh/community/forums/topics/982516) |

#### [Pending Cup \#8 (2020)](https://osu.ppy.sh/community/forums/topics/1145075)

> | Song | Mode | ![Gold Crown][GCrown] | ![Silver Crown][SCrown] | ![Bronze Crown][BCrown] | Results |

Reviewer: despite all the winners using the same song, each of the category uses two songs. i'd probably change it to be like NPC 6

# Contests

> Section dedicated for contests held throughout the osu! community.

Author: oh this sentence is actually modelled after http://osu.ppy.sh/wiki/en/Tournaments's opening sentence but yeah i think i'll go with yours as it feels much better

| [#4](https://osu.ppy.sh/home/news/2018-08-19-aspire-2018-begins) | [Culprate - Acid Rain](https://osu.ppy.sh/beatmaps/artists/29) | ![][flag_TW] [SnowNiNo\_](https://osu.ppy.sh/users/2506267) | ![][flag_FI] [DTM9 Nowa](https://osu.ppy.sh/users/5428909) | ![][flag_LT] [seselis1](https://osu.ppy.sh/users/7321833) | [Results](https://osu.ppy.sh/home/news/2019-07-22-aspire-2018-results) |
| [#5](https://osu.ppy.sh/home/news/2020-04-09-aspire-v-a-new-format) | Any song available in the [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists) as per April 2020 | ![][flag_NL] [Mr HeliX](https://osu.ppy.sh/users/2330619) *(community pick winner)*, ![][flag_CN] [fanzhen0019](https://osu.ppy.sh/users/418699) *(mapper's pick winner)* | ![][flag_CA] [Meow Mix](https://osu.ppy.sh/users/3021634) | ![][flag_CN] [fanzhen0019](https://osu.ppy.sh/users/418699) | [Results](https://osu.ppy.sh/home/news/2020-08-23-aspire-v-results) |

#### osu! Mapping Olympiad

*Main page: [osu! Mapping Olympiad](/wiki/Contests/Mapping_Olympiad)*

> The *osu! Mapping Olympiad* is a beatmapping contest that is held as a spiritual successor to the earlier (pre-2020) Monthly Beatmapping Contests. In this contest, participating mappers are tasked to create either a full beatmapset (with a proper [difficulty spread](wiki/Beatmap/Difficulty#difficulty-levels) starting from Easy or Normal onwards) or a certain number of difficulties for any of the given songs in each corresponding iteration.

Reviewer 2: ```suggestion
The **osu! Mapping Olympiad** is a mapping contest that is held as a spiritual successor to the earlier (pre-2020) Monthly Beatmapping Contests. In this contest, participating mappers are tasked to create either a beatmap with a proper [difficulty spread](wiki/Beatmap/Difficulty#difficulty-levels) starting from Easy or Normal onwards or a certain number of difficulties for any of the given songs in each corresponding iteration.
```

| [#7](https://osu.ppy.sh/home/news/2020-06-06-osu-mapping-olympiad-7-mania) | [Frums - Star Of The COME ON!!](https://osu.ppy.sh/beatmaps/artists/68), [IAHN - Welcome (Original Mix)](https://osu.ppy.sh/beatmaps/artists/3), [PUP - DVP](https://osu.ppy.sh/beatmaps/artists/79), [HyuN - ENCOUNTER](https://osu.ppy.sh/beatmaps/artists/33), and [Cranky - Ran](https://osu.ppy.sh/beatmaps/artists/23) | ![osu!mania][osu!mania] | ![][flag_US] [Toaph Daddy](https://osu.ppy.sh/users/7616811) and ![][flag_KR] [Kawawa](https://osu.ppy.sh/users/4647754) | *N/A* | ![][flag_PH] [lenpai](https://osu.ppy.sh/users/5314573) and ![][flag_CA] [BringoBrango](https://osu.ppy.sh/users/10274043) | [Results](https://osu.ppy.sh/home/news/2020-12-01-osu-mapping-olympiad-7-concludes) |

#### Community Mapping Contest

> he *Community Mapping Contest* is a beatmapping contest where participating mappers are free to submit their entry for any difficulty in any game mode they desire, after which all winning entries across all difficulties and game modes will then be merged together into a single beatmap.

Reviewer 2: ```suggestion
The **Community Mapping Contest** is a mapping contest where participating mappers are free to submit their entry for any difficulty in any game mode they desire, after which all winning entries across all difficulties and game modes will then be merged together into a single beatmap.
```

| [#1 (osu!mania (4K))](https://osu.ppy.sh/home/news/2016-09-28-community-mapping-contest-1-yuki-nadeshiko) | [yuki. - Nadeshiko Sensation](https://osu.ppy.sh/beatmaps/artists/4) | ![osu!mania][osu!mania] | ![][flag_CN] [Kayano](https://osu.ppy.sh/users/961752) | ![][flag_ID] [LordRaika](https://osu.ppy.sh/users/3265023) | ![][flag_ID] [LordRaika](https://osu.ppy.sh/users/3265023) | ![][flag_FR] [Ovoui](https://osu.ppy.sh/users/2282098) | ![][flag_ID] [DoNotMess](https://osu.ppy.sh/users/1596318) | [Results](https://osu.ppy.sh/beatmapsets/587388) |
| [#1 (osu!mania (7K))](https://osu.ppy.sh/home/news/2016-09-28-community-mapping-contest-1-yuki-nadeshiko) | [yuki. - Nadeshiko Sensation](https://osu.ppy.sh/beatmaps/artists/4) | ![osu!mania][osu!mania] | ![][flag_VN] [Kuo Kyoka](https://osu.ppy.sh/users/2596306) | ![][flag_VN] [Kuo Kyoka](https://osu.ppy.sh/users/2596306) | ![][flag_VN] [Kuo Kyoka](https://osu.ppy.sh/users/2596306) | ![][flag_VN] [Kuo Kyoka](https://osu.ppy.sh/users/2596306) | ![][flag_VN] [Kuo Kyoka](https://osu.ppy.sh/users/2596306) | [Results](https://osu.ppy.sh/beatmapsets/587388) |

#### Mapping With Rewards

> *Mapping With Rewards* is a beatmapping contest where participating mappers compete to create the best beatmap (with a proper [difficulty spread](wiki/Beatmap/Difficulty#difficulty-levels) starting from Easy or Normal onwards) for any of the given songs in each corresponding iteration without strict deadlines.

Reviewer 2: ```suggestion
**Mapping With Rewards** is a mapping contest where participating mappers compete to create the best beatmap (with a proper [difficulty spread](wiki/Beatmap/Difficulty#difficulty-levels) starting from Easy or Normal onwards) for any of the given songs in each corresponding iteration without strict deadlines.
```

| [#3.2 (osu!taiko)](https://osu.ppy.sh/home/news/2017-09-23-mapping-with-rewards-returns) | [\*namirin - closing eyes](https://osu.ppy.sh/beatmaps/artists/18) | ![osu!taiko][osu!taiko] | ![][flag_HK] [Nardoxyribonucleic](https://osu.ppy.sh/users/876419) | ![][flag_US] [Jonarwhal](https://osu.ppy.sh/users/3653035) | *N/A* | [Results](https://osu.ppy.sh/home/news/2020-05-11-mappers-report-april) |
| [#3.3](https://osu.ppy.sh/home/news/2017-09-23-mapping-with-rewards-returns) | [Fractal Dreamers - Fortuna Redux](https://osu.ppy.sh/beatmaps/artists/15) | ![osu!][osu!] | ![][flag_FI] [DTM9 Nowa](https://osu.ppy.sh/users/5428909) | ![][flag_MY] [HootOwlStar](https://osu.ppy.sh/users/4341302) | *N/A* | [Results](https://osu.ppy.sh/home/news/2020-05-11-mappers-report-april) |

### Other official contests

#### A Labour of Love

> *A Labour of Love* is a team-based contest spanning over multiple artistic disciplines where participating teams are tasked to remix/rearrange a song that has been widely known throughout the osu! community, create a beatmap out of the remixed song, and design an original storyboard/background video as a visual accompaniment to said beatmap. As [indicated by this tweet](https://twitter.com/ppy/status/1333362145224843264 "Twitter"), the contest is modeled after [THE BMS OF FIGHTERS events](https://www.bmsoffighters.net/ "BMS OF FIGHTERS") and other similar competitions.

Reviewer 2: ```suggestion
**A Labour of Love** is a team-based contest spanning over multiple artistic disciplines where participating teams are tasked to remix/rearrange a song that has been widely known throughout the osu! community, create a beatmap out of the remixed song, and design an original storyboard/background video as a visual accompaniment to said beatmap. As [indicated by this tweet](https://twitter.com/ppy/status/1333362145224843264 "Twitter"), the contest is modeled after [THE BMS OF FIGHTERS events](https://www.bmsoffighters.net/ "BMS OF FIGHTERS") and other similar competitions.
```

| :-: | :-- | :-- | :-- | :-- | :-- |
| [1](https://osu.ppy.sh/home/news/2020-11-30-a-labour-of-love) | [Kuba Oms - My Love](https://osu.ppy.sh/beatmaps/artists/13) | extremely valid (![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471), ![][flag_FR] [Shinwoir](https://osu.ppy.sh/users/8984574), ![][flag_DE] [Mao](https://osu.ppy.sh/users/2204515), ![][flag_FR] [Realazy](https://osu.ppy.sh/users/918297), and ![][flag_GB] [DeviousPanda](https://osu.ppy.sh/users/4966334)) | schoolboy fans (![][flag_RU] [Shadren](https://osu.ppy.sh/users/745494), ![][flag_UA] [wajinshu](https://osu.ppy.sh/users/6339790), ![][flag_RU] [Shmiklak](https://osu.ppy.sh/users/5504231), and ![][flag_RU] [Daycore](https://osu.ppy.sh/users/5596337)) | Team Red but Blue (![][flag_CA] [Redside](https://osu.ppy.sh/users/16039046), ![][flag_US] [Tofumang](https://osu.ppy.sh/users/4817223), ![][flag_CA] [Chompy](https://osu.ppy.sh/users/7427035), ![][flag_US] [Noffy](https://osu.ppy.sh/users/1541323), and ![][flag_CA] [Zer0-G](https://osu.ppy.sh/users/12577911)) | [Results](https://osu.ppy.sh/home/news/2021-04-27-results-a-labour-of-love) |

#### Best of/Community Choice

> he *Best of/Community Choice* contest is a public voting contest that is being held annually to determine the most popular beatmaps <!-- https://osu.ppy.sh/home/news/2021-02-05-community-choice-2020-results --> in the eyes of the osu! community at each given year.

Reviewer 2: ```suggestion
The **Best of/Community Choice** contest is a public voting contest that is being held annually to determine the most popular beatmaps in the eyes of the osu! community at each given year.
```

Author: oh for this one (as i already explained in a prior review) the comment is actually intentional as that news post is the one news post that explicitly states that the contest is designated to search for "the most popular" despite the contest's name.

it's there as a reference to future editors if any...

> At its launch, the contest was named "Best of" as the initial premise and expectation of the contest was to encourage members of the community to vote for what they consider as "the best beatmaps of the year"; however, as the game's playerbase continue to grow in number over time, the contest was inevitably shifting more and more into a popularity contest each year which prompted the contest to be relaunched as "Community Choice" in 2017.

Reviewer 2: ```suggestion
At its launch, the contest was named "Best of", as the initial premise and expectation of the contest was to encourage members of the community to vote for what they consider as "the best beatmaps of the year"; however, as the game's playerbase continued to grow in number over time, the contest was inevitably shifting more and more into a popularity contest each year which prompted the contest to be relaunched as "Community Choice" in 2017.
```

| [2020](https://osu.ppy.sh/home/news/2021-01-21-community-choice-2020-voting-open) | ![osu!][osu!]/![osu!taiko][osu!taiko]/![osu!catch][osu!catch]/![osu!mania][osu!mania] | [DADDY ! DADDY ! DO ! feat. Suzuki Airi (TV Size)](https://osu.ppy.sh/beatmapsets/1147361) hosted by ![][flag_GB] [hypercyte](https://osu.ppy.sh/users/9155377) | [Calliope Mori - Excuse My Rudeness, but Could You Please RIP?<3 (t+pazolite Remix)](https://osu.ppy.sh/beatmapsets/1278695) hosted by ![][flag_DE] [Capu](https://osu.ppy.sh/users/2474015) | [Camellia feat. Nanahira - finorza](https://osu.ppy.sh/beatmapsets/1151836) hosted by ![][flag_KR] [Rocma](https://osu.ppy.sh/users/566276) | [Camellia - We Could Get More Machinegun Psystyle! (And More Genre Switches)](https://osu.ppy.sh/beatmapsets/1034114) hosted by ![][flag_GB] [Hydria](https://osu.ppy.sh/users/808176) | [Results](https://osu.ppy.sh/home/news/2021-02-05-community-choice-2020-results) |

#### Fan Art Contest

> The *Fan Art Contest* is a drawing contest where participating artists from all across the community are challenged to create the best fan art for osu! involving [osu!'s mascots](wiki/Mascots) across various themes and art concepts.

Reviewer 2: ```suggestion
The **Fan Art Contest** is a drawing contest where participating artists from the community are challenged to create the best fan art for osu! involving [osu!'s mascots](wiki/Mascots) across various themes and art concepts.
```

| [#25 (Spring 2021)](https://osu.ppy.sh/home/news/2021-04-23-spring-2021-fanart-contest) | A Day In The Park | ![][flag_US] [Tofumang](https://osu.ppy.sh/users/4817223) | ![][flag_MY] [Kyvrin](https://osu.ppy.sh/users/11589256) | ![][flag_BG] [risuuna](https://osu.ppy.sh/users/7266506) | [Results](https://osu.ppy.sh/home/news/2021-05-20-spring-fanart-contest-results) |

#### osu! Beatmap Blueprints Contest

The *osu! Beatmap Blueprints Contest* is a one-off contest held in commemoration of the launch of [Beatmap Blueprints](https://osu.ppy.sh/store/products/9) in the [osu!store](https://osu.ppy.sh/store/listing). In this contest, participating users competed to create the best blueprint of a beatmap in any visual medium using all the assets provided in the Beatmap Blueprints merchandise pack.

| Contest No. | ![Gold Crown][GCrown] | ![Silver Crown][SCrown] | ![Bronze Crown][BCrown] | Results |
| :-: | :-- | :-- | :-- | :-- |
| [#1](https://osu.ppy.sh/home/news/2015-03-20-osu-beatmap-blueprints-available-contest) | *N/A* | *N/A* | *N/A* | *N/A* |

#### osu!idol

> *osu!idol* is a singing/song cover contest where participants from all across the community compete to create the best song cover (either solo or with a partner) across multiple judging and community voting stages. The contest is named after the [Idols franchise](https://en.wikipedia.org/wiki/Idols_(franchise) "Wikipedia") and is mainly inspired by various televised singing competition shows.

Reviewer 2: ```suggestion
**osu!idol** is a singing contest where participants from all across the community compete to create the best song cover (either solo or with a partner) across multiple judging and community voting stages. The contest is named after the [Idols franchise](https://en.wikipedia.org/wiki/Idols_(franchise) "Wikipedia") and is mainly inspired by various televised singing competition shows.
```

| [#5 (2018)](https://osu.ppy.sh/home/news/2018-07-01-osu-idol-2018-auditions-now-open) | ![][flag_US] [Will Stetson](https://osu.ppy.sh/users/4909088) | ![][flag_ID] [Zeva\[FL\]](https://osu.ppy.sh/users/2805170) | ![][flag_US] [MjIsMaster](https://osu.ppy.sh/users/9922896) | [Will Stetson - Snow Drive](https://assets.ppy.sh/contests/71/entries/Snow%20Drive.mp3) | [Results](https://osu.ppy.sh/home/news/2018-11-05-osu-idol-2018-finals-community-voting-results) |
| [#6 (2019)](https://osu.ppy.sh/home/news/2019-07-02-osu-idol-2019-auditions-now-open) | ![][flag_PH] [iamALieee](https://osu.ppy.sh/users/10093980) | ![][flag_US] [aceredshirt13](https://osu.ppy.sh/users/13736238) | ![][flag_US] [moof](https://osu.ppy.sh/users/8457202) | [iamALieee - White First Love](https://assets.ppy.sh/contests/83/entries/osu!idol%202019%20-%20iamALieee%20-%20White%20First%20Love.mp3) | [Results](https://osu.ppy.sh/home/news/2019-12-21-osu-idol-2019-finals-community-voting-results) |
| [#7 (2020)](https://osu.ppy.sh/home/news/2020-09-01-osu-idol-2020-auditions-now-open) | *N/A* | *N/A* | *N/A* | *N/A* | *N/A* |

#### osu!remix

> *osu!remix* is a song remix contest where participating musicians from all across the community are tasked to create the best remix of a chosen song that has been widely known throughout the osu! community in one way or another. It is the first ever contest to be handled via the [Community Contests listing](https://osu.ppy.sh/community/contests) page.

Reviewer 2: ```suggestion
**osu!remix** is a song remix contest where participating musicians from all across the community are tasked to create the best remix of a chosen song that has been widely known throughout the osu! community in one way or another. It is the first ever contest to be handled via the [Contests listing](https://osu.ppy.sh/community/contests) page.
```

| Contest No. | Remixed Song | ![Gold Crown][GCrown] | ![Silver Crown][SCrown] | ![Bronze Crown][BCrown] | Results |
| :-: | :-- | :-- | :-- | :-- | :-- |
| [#1](https://osu.ppy.sh/home/news/2016-05-06-osu-circles-remix-contest) | [nekodex - circles!](https://osu.ppy.sh/beatmaps/artists/1) | ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471) | ![][flag_FR] [Kurokotei](https://osu.ppy.sh/users/398275) | ![][flag_NL] [happy30](https://osu.ppy.sh/users/27767) | [Results](https://osu.ppy.sh/home/news/2016-08-29-osuremix-contest-1-results) |

#### Skinning Contest

The Skinning Contest is a skinning contest open to skin creators from all across the community. 

| Contest No. | ![Gold Crown][GCrown] | ![Silver Crown][SCrown] | ![Bronze Crown][BCrown] | Results |
| :-: | :-- | :-- | :-- | :-- |

> | [#1](wiki/Contests/Skinning_Contest/) | [FREEDOM DiVE↓](https://osu.ppy.sh/community/forums/topics/1293207) by ![][flag_US] [BTMC](https://osu.ppy.sh/users/3171691) | [Minimalist 2.0](https://osu.ppy.sh/community/forums/topics/1286223) by ![][flag_EE] [Krizto](https://osu.ppy.sh/users/13474556) | [notMiku](https://osu.ppy.sh/community/forums/topics/1292954) by ![][flag_RU] [NikStrim](https://osu.ppy.sh/users/10577559) | [Results](https://osu.ppy.sh/home/news/2021-05-27-skinning-contest-results) |

Reviewer 2: ```suggestion
| [#1](/wiki/Contests/Skinning_Contest/1) | [FREEDOM DiVE↓](https://osu.ppy.sh/community/forums/topics/1293207) by ![][flag_US] [BTMC](https://osu.ppy.sh/users/3171691) | [Minimalist 2.0](https://osu.ppy.sh/community/forums/topics/1286223) by ![][flag_EE] [Krizto](https://osu.ppy.sh/users/13474556) | [notMiku](https://osu.ppy.sh/community/forums/topics/1292954) by ![][flag_RU] [NikStrim](https://osu.ppy.sh/users/10577559) | [Results](https://osu.ppy.sh/home/news/2021-05-27-skinning-contest-results) |
```

> ## Unofficial/community-run contests

Reviewer 2: ```suggestion
## Community-run contests
```

> *Unofficial contests* are contests that are hosted by members of the community. Contest organizers who are looking to add their contest to the list below are welcome to do so by opening a GitHub pull request in the [osu-wiki repository](https://github.com/ppy/osu-wiki "GitHub") following the [osu! wiki contribution guide](/wiki/osu!_wiki_Contribution_Guide).

Reviewer 2: ```suggestion
*Unofficial contests* are contests that are hosted by the members of the community.
```
i feel like this should be a part of a seperate README, not the article

Author: okok

> ### osu! Beatmapping World Championship <!-- Yes, OBWC is not an official contest according to https://twitter.com/paichiru/status/1408900730149249025 -->

Reviewer 2: ```suggestion
### osu! Beatmapping World Championship
```

Author: yeah i believe this comment is also supposed to be there to avoid future editors from being confused as to "why is this not listed as official"...

> *osu! Beatmapping World Championship* is a team-based beatmapping contest where participating teams consisting of mappers from the same country, region, or even continent (to some extent) battle their beatmapping skills out against each other across multiple songs and mapping stages. The contest is known for exclusively making use of songs available in the [Featured Artist](wiki/Featured_Artists) library during its runs.

Reviewer 2: ```suggestion
**osu! Beatmapping World Championship** is a team-based mapping contest where participating teams consisting of mappers from the same country, region, or even continent (to some extent) battle their mapping skills out against each other across multiple songs and mapping stages. The contest is known for exclusively making use of songs available in the [Featured Artist](wiki/Featured_Artists) library during its runs.
```

| [#2 (2020)](/wiki/Contests/oBWC/2) | ![][flag_GB] United Kingdom ([hypercyte](https://osu.ppy.sh/users/9155377), [Aistre](https://osu.ppy.sh/users/4879380), [Altai](https://osu.ppy.sh/users/5745865), [Chugger](https://osu.ppy.sh/users/4491713), [DeviousPanda](https://osu.ppy.sh/users/4966334), [pocket-](https://osu.ppy.sh/users/6808091), and [Yusomi](https://osu.ppy.sh/users/4174940)) | ![][flag_AR] Argentina ([MaestroSplinter](https://osu.ppy.sh/users/6707918), [chucentry](https://osu.ppy.sh/users/2498731), [Lince Cosmico](https://osu.ppy.sh/users/6070370), and [Megafan](https://osu.ppy.sh/users/6632605)) | ![][flag_BR] Brazil ([Net0](https://osu.ppy.sh/users/5099768), [Enerugi](https://osu.ppy.sh/users/4475985), [Faito](https://osu.ppy.sh/users/9706291), [Kowari](https://osu.ppy.sh/users/5404892), [Maot](https://osu.ppy.sh/users/3914271), [Sakura Airi](https://osu.ppy.sh/users/8682057), and [Trynna](https://osu.ppy.sh/users/2652951)) | [Results](https://osu.ppy.sh/home/news/2020-10-22-beatmapping-world-championship-2020-concludes) |

### Pending Cup

> The *Pending Cup* is a beatmapping contest that is held yearly by prominent members of the Chinese mapping community. Each year, the contest features up to three songs and/or categories that mappers from all across the community are free to compete and submit their entries in.

Reviewer 2: ```suggestion
The **Pending Cup** is a mapping contest that is held yearly by prominent members of the Chinese mapping community. Each year, the contest features up to three songs and/or categories that mappers from all across the community are free to compete and submit their entries in.
```

| Vocal² | ![osu!][osu!] | ![][flag_KR] [Heilia](https://osu.ppy.sh/users/9823042) | ![][flag_DE] [jamesjan3](https://osu.ppy.sh/users/6260705) | ![][flag_CN] [Garden](https://osu.ppy.sh/users/2849992) | [Results](https://osu.ppy.sh/community/forums/topics/1198663) |
| Instrumental³ | ![osu!][osu!] | ![][flag_RU] [xbopost](https://osu.ppy.sh/users/6842421) | ![][flag_ID] [\[Keqing\]](https://osu.ppy.sh/users/8972308) | ![][flag_KR] [Acylica](https://osu.ppy.sh/users/1943309) | [Results](https://osu.ppy.sh/community/forums/topics/1198663) |

¹ The Featured Artist category provides mappers with a choice of two different songs to be mapped: Zekk - Re\_construct and Shuu no Hazama \[Rainshower\].

² The Vocal category provides mappers with a choice of two different songs to be mapped: nonoc - memento and DUSTCELL - DERO.

³ The Instrumental category provides mappers with a choice of two different songs to be mapped: onoken - Felys final remix and Nekomata Master - The Sky of Sadness.

### Newspaper Cup

> The *Newspaper Cup* is a beatmapping contest that is similar to the Pending Cup in nature but with a much greater emphasis on novice mappers. As with Pending Cup, the contest is also being held yearly by prominent members of the Chinese mapping community and has since been developed from a contest that was particularly intended for the Chinese-speaking members of the community to a global-scale contest.

Reviewer 2: ```suggestion
The **Newspaper Cup** is a mapping contest that is similar to the Pending Cup in nature but with a much greater emphasis on novice mappers. As with Pending Cup, the contest is also being held yearly by prominent members of the Chinese mapping community and has since been developed from a contest that was particularly intended for the Chinese-speaking members of the community to a global-scale contest.
```

# Contests

Section dedicated for contests held throughout the osu! community.

> ## Official Contests

Reviewer 3: Headers should be in sentence case

> Official contests are contests that are being hosted (or otherwise recognized as official) by the [osu!team](wiki/People/The_Team). Most official contests have their own listing in the [contests page](https://osu.ppy.sh/community/contests) dedicated to them.

Reviewer 3: ```suggestion
Official contests are contests that are hosted (or otherwise recognised as official) by the [osu! team](wiki/People/The_Team). Most official contests have their own listing in the [contests page](https://osu.ppy.sh/community/contests) dedicated to them.
```

### Official Beatmapping Contests

#### Monthly Beatmapping Contest

*Main page: [Monthly Beatmapping Contest](/wiki/Contests/Monthly_Beatmapping_Contest)*

> The Monthly Beatmaping Contest in its initial concept was a beatmapping contest where participating mappers were competing to create the most excellent mapset with a proper [difficulty spread](wiki/Beatmap/Difficulty#difficulty-levels) starting from Easy or Normal onwards of a given song (that had been cleared for use in osu! preemptively by the [osu!team](wiki/People/The_Team)) in a month's time. 

Reviewer 3: ```suggestion
The *Monthly Beatmaping Contest* is a beatmapping contest where participating mappers compete to create the most excellent beatmapset with a proper [difficulty spread](wiki/Beatmap/Difficulty#difficulty-levels) starting from Easy or Normal onwards of a given song in a month's time. 
```
tense should remain consistent throughout; this sounds a bit awkward

AFAICT the first paragraph should just describe what it generally *is*, and then following ones can explain differences between previous and current versions and other minutia

Author: given the context of this sentence i guess it should all be written in past tense, yeah ;~;

| [#12](https://osu.ppy.sh/home/news/2016-03-04-beatmapping-contest-12-osu) | IAHN - Candy Luv | ![osu!][osu!] | ![][flag_KR] [Taeyang](https://osu.ppy.sh/users/2732340) | ![][flag_CA] [Monstrata](https://osu.ppy.sh/users/2706438) | ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) | [Results](https://osu.ppy.sh/home/news/2016-05-26-monthly-beatmapping-contest-12-results) |
| [#13](https://osu.ppy.sh/home/news/2016-12-09-monthly-beatmapping-contest-13-now-open) | BilliumMoto - HDHR | ![osu!][osu!] | ![][flag_MY] [RyoKazuka](https://osu.ppy.sh/users/6258586) | ![][flag_ID] [Fort](https://osu.ppy.sh/users/2805457) | ![][flag_DE] [Pho](https://osu.ppy.sh/users/3624692) | [Results](https://osu.ppy.sh/home/news/2017-04-02-monthly-beatmapping-contest-13-results) |

> After a long hiatus, the Monthly Beatmapping Contest is being rebooted in January 2020 with a brand new schedule, ruleset, voting procedure, and reward scheme. Many aspects of the competition have been dramatically altered to make way for a more engaging and attractive contest, in which some of them are as follows:

Reviewer 3: ```suggestion
After a long hiatus, the Monthly Beatmapping Contest was rebooted in January 2020 with a brand new schedule, rule set, voting procedure, and reward scheme. Many aspects of the competition were dramatically altered to make way for a more engaging and attractive contest:
```
may be worth mentioning when that hiatus was

Author: think i'll keep the "in which some of them are as follows:" here as the list only covers some of the main changes (or else the list would be unnecessarily long for what is supposed to be an index page of some sort lol)

applied everything else otherwise

> - Instead of a single song, the contest now offers a selection of several songs from a different [featured artist](/wiki/Featured_Artists) each month in which all participating mappers are free to map and submit their entries from.
> - Participating mappers now only have to submit a single difficulty instead of a complete mapset.
> - Participating mappers now have to adhere to certain beatmapping theme and/or limitation set by the contest each month, such as "Star rating may not exceed 4.00★" or "Every object should overlap another object".

Reviewer 3: ```suggestion
- Instead of a single song, the contest now offers a selection of several songs from a different [featured artist](/wiki/Featured_Artists) each month where all participating creators are free to map and submit their entries from.
- Participating creators now only have to submit a single difficulty instead of a complete mapset.
- Participating creators now have to adhere to certain beatmapping theme and/or limitation set by the contest each month, such as "Star rating may not exceed 4.00★" or "Every object should overlap another object".
```
tense still sounds a bit off. should probably do a more thorough check on the whole article to make sure it's consistent

also, for some reason, the [ASC wants articles to use the term "creator"](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Writing#terminology) instead of "mapper" or "beatmapper". Should probably go through and ctrl-f this

Author: mmmmmmmmmm really not sure about this as the term "creator" is, as far as i'm aware of, very very very very very rarely used in the mapping environment at large to refer to a mapper/beatmapper.

might get more info on this

> For more information about how the current Monthly Beatmapping Contest cycles are being run, please refer to their [main article](/wiki/Contests/Monthly_Beatmapping_Contest).

Reviewer 3: this is unnecessary; main page article does this job

| [April 2021 (osu!taiko)](https://osu.ppy.sh/community/contests/123) |  [MYUKKE.](https://osu.ppy.sh/beatmaps/artists/121) | ![osu!taiko][osu!taiko] | ![][flag_TH] [KappuChinooo](https://osu.ppy.sh/users/9582525) | ![][flag_JP] [uone](https://osu.ppy.sh/users/5321719) | *N/A* | [Results](https://osu.ppy.sh/home/news/2021-06-25-results-monthly-beatmapping-contest-may-osu) |
| [April 2021 (osu!catch)](https://osu.ppy.sh/community/contests/124) |  [MYUKKE.](https://osu.ppy.sh/beatmaps/artists/121) | ![osu!catch][osu!catch] | ![][flag_HK] [autofanboy](https://osu.ppy.sh/users/636114) | ![][flag_TH] [rew0825](https://osu.ppy.sh/users/2488026) | *N/A* | [Results](https://osu.ppy.sh/home/news/2021-06-25-results-monthly-beatmapping-contest-may-osu) |
| [May 2021 (osu!)](https://osu.ppy.sh/community/contests/127) |  [Masahiro "Godspeed" Aoki](https://osu.ppy.sh/beatmaps/artists/136) | ![osu!][osu!] | ![][flag_GB] [DeviousPanda](https://osu.ppy.sh/users/4966334) | ![][flag_RU] [kuyusu](https://osu.ppy.sh/users/11758667) | ![][flag_RU] [kuyusu](https://osu.ppy.sh/users/11758667) | [Results](https://osu.ppy.sh/home/news/2021-06-25-results-monthly-beatmapping-contest-may-osu) |

#### Aspire

> Aspire is a one-of-a-kind beatmapping contest where participating mappers are being challenged to create the most innovative beatmap imaginable without any restrictions towards the rules of the [Ranking Criteria](/wiki/Ranking_Criteria). The contest encourages mappers to explore the the technical possibilities of the game as far as possible.

Reviewer 3: ```suggestion
*Aspire* is a one-of-a-kind beatmapping contest where participating creators are challenged to create the most innovative beatmap imaginable without any restrictions towards the rules of the [Ranking Criteria](/wiki/Ranking_Criteria). The contest encourages creators to explore the the technical possibilities of the game as far as possible.
```

| [#4](https://osu.ppy.sh/home/news/2018-08-19-aspire-2018-begins) | [Culprate - Acid Rain](https://osu.ppy.sh/beatmaps/artists/29) | ![][flag_TW] [SnowNiNo\_](https://osu.ppy.sh/users/2506267) | ![][flag_FI] [DTM9 Nowa](https://osu.ppy.sh/users/5428909) | ![][flag_LT] [seselis1](https://osu.ppy.sh/users/7321833) | [Results](https://osu.ppy.sh/home/news/2019-07-22-aspire-2018-results) |
| [#5](https://osu.ppy.sh/home/news/2020-04-09-aspire-v-a-new-format) | Any song available in the [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists) as per April 2020 | ![][flag_NL] [Mr HeliX](https://osu.ppy.sh/users/2330619) *(community pick winner)*, ![][flag_CN] [fanzhen0019](https://osu.ppy.sh/users/418699) *(mapper's pick winner)* | ![][flag_CA] [Meow Mix](https://osu.ppy.sh/users/3021634) | ![][flag_CN] [fanzhen0019](https://osu.ppy.sh/users/418699) | [Results](https://osu.ppy.sh/home/news/2020-08-23-aspire-v-results) |

#### osu! Mapping Olympiad

*Main page: [osu! Mapping Olympiad](/wiki/Contests/Mapping_Olympiad)*

> The osu! Mapping Olympiad is a beatmapping contest that is being held as a spiritual successor to the earlier (pre-2020) Monthly Beatmapping Contests. In this contest, participating mappers are being tasked to create either a full mapset (with a proper [difficulty spread](wiki/Beatmap/Difficulty#difficulty-levels) starting from Easy or Normal onwards) or a certain number of difficulties for any of the given songs in each corresponding iteration.

Reviewer 3: ```suggestion
The *osu! Mapping Olympiad* is a beatmapping contest that is held as a spiritual successor to the earlier (pre-2020) Monthly Beatmapping Contests. In this contest, participating creators are tasked to create either a full beatmapset (with a proper [difficulty spread](wiki/Beatmap/Difficulty#difficulty-levels) starting from Easy or Normal onwards) or a certain number of difficulties for any of the given songs in each corresponding iteration.
```

| [#7](https://osu.ppy.sh/home/news/2020-06-06-osu-mapping-olympiad-7-mania) | [Frums - Star Of The COME ON!!](https://osu.ppy.sh/beatmaps/artists/68), [IAHN - Welcome (Original Mix)](https://osu.ppy.sh/beatmaps/artists/3), [PUP - DVP](https://osu.ppy.sh/beatmaps/artists/79), [HyuN - ENCOUNTER](https://osu.ppy.sh/beatmaps/artists/33), and [Cranky - Ran](https://osu.ppy.sh/beatmaps/artists/23) | ![osu!mania][osu!mania] | ![][flag_US] [Toaph Daddy](https://osu.ppy.sh/users/7616811) and ![][flag_KR] [Kawawa](https://osu.ppy.sh/users/4647754) | *N/A* | ![][flag_PH] [lenpai](https://osu.ppy.sh/users/5314573) and ![][flag_CA] [BringoBrango](https://osu.ppy.sh/users/10274043) | [Results](https://osu.ppy.sh/home/news/2020-12-01-osu-mapping-olympiad-7-concludes) |

#### Community Mapping Contest

> The Community Mapping Contest is a beatmapping contest where participating mappers are free to submit their entry for any difficulty in any game mode they desire, in which all the winning entries across all difficulties and game modes are then merged together into a single beatmap.

Reviewer 3: ```suggestion
The *Community Mapping Contest* is a beatmapping contest where participating mappers are free to submit their entry for any difficulty in any game mode they desire, after which all winning entries across all difficulties and game modes will then be merged together into a single beatmap.
```

| [#1 (osu!mania (4K))](https://osu.ppy.sh/home/news/2016-09-28-community-mapping-contest-1-yuki-nadeshiko) | [yuki. - Nadeshiko Sensation](https://osu.ppy.sh/beatmaps/artists/4) | ![osu!mania][osu!mania] | ![][flag_CN] [Kayano](https://osu.ppy.sh/users/961752) | ![][flag_ID] [LordRaika](https://osu.ppy.sh/users/3265023) | ![][flag_ID] [LordRaika](https://osu.ppy.sh/users/3265023) | ![][flag_FR] [Ovoui](https://osu.ppy.sh/users/2282098) | ![][flag_ID] [DoNotMess](https://osu.ppy.sh/users/1596318) | [Results](https://osu.ppy.sh/beatmapsets/587388) |
| [#1 (osu!mania (7K))](https://osu.ppy.sh/home/news/2016-09-28-community-mapping-contest-1-yuki-nadeshiko) | [yuki. - Nadeshiko Sensation](https://osu.ppy.sh/beatmaps/artists/4) | ![osu!mania][osu!mania] | ![][flag_VN] [Kuo Kyoka](https://osu.ppy.sh/users/2596306) | ![][flag_VN] [Kuo Kyoka](https://osu.ppy.sh/users/2596306) | ![][flag_VN] [Kuo Kyoka](https://osu.ppy.sh/users/2596306) | ![][flag_VN] [Kuo Kyoka](https://osu.ppy.sh/users/2596306) | ![][flag_VN] [Kuo Kyoka](https://osu.ppy.sh/users/2596306) | [Results](https://osu.ppy.sh/beatmapsets/587388) |

#### Mapping With Rewards

> Mapping With Rewards is a beatmapping contest where participating mappers are competing to create the best beatmap (with a proper [difficulty spread](wiki/Beatmap/Difficulty#difficulty-levels) starting from Easy or Normal onwards) for any of the given songs in each corresponding iteration without strict deadlines.

Reviewer 3: ```suggestion
*Mapping With Rewards* is a beatmapping contest where participating creators compete to create the best beatmap (with a proper [difficulty spread](wiki/Beatmap/Difficulty#difficulty-levels) starting from Easy or Normal onwards) for any of the given songs in each corresponding iteration without strict deadlines.
```

| [#3.2 (osu!taiko)](https://osu.ppy.sh/home/news/2017-09-23-mapping-with-rewards-returns) | [\*namirin - closing eyes](https://osu.ppy.sh/beatmaps/artists/18) | ![osu!taiko][osu!taiko] | ![][flag_HK] [Nardoxyribonucleic](https://osu.ppy.sh/users/876419) | ![][flag_US] [Jonarwhal](https://osu.ppy.sh/users/3653035) | *N/A* | [Results](https://osu.ppy.sh/home/news/2020-05-11-mappers-report-april) |
| [#3.3](https://osu.ppy.sh/home/news/2017-09-23-mapping-with-rewards-returns) | [Fractal Dreamers - Fortuna Redux](https://osu.ppy.sh/beatmaps/artists/15) | ![osu!][osu!] | ![][flag_FI] [DTM9 Nowa](https://osu.ppy.sh/users/5428909) | ![][flag_MY] [HootOwlStar](https://osu.ppy.sh/users/4341302) | *N/A* | [Results](https://osu.ppy.sh/home/news/2020-05-11-mappers-report-april) |

### Other Official Contests

#### A Labour of Love

> A Labour of Love is a team-based spanning over multiple artistic disciplines where participating teams are being tasked to remix/rearrange a song that has been widely known throughout the osu! community, create a beatmap out of the remixed song, and design an original storyboard/background video as a visual accompaniment to the said beatmap. As [indicated by this tweet](https://twitter.com/ppy/status/1333362145224843264), the contest is modelled after [THE BMS OF FIGHTERS events](https://www.bmsoffighters.net/) and other similar competitions.

Reviewer 3: ```suggestion
*A Labour of Love* is a team-based contest spanning over multiple artistic disciplines where participating teams are tasked to remix/rearrange a song that has been widely known throughout the osu! community, create a beatmap out of the remixed song, and design an original storyboard/background video as a visual accompaniment to said beatmap. As [indicated by this tweet](https://twitter.com/ppy/status/1333362145224843264 "Twitter"), the contest is modeled after [THE BMS OF FIGHTERS events](https://www.bmsoffighters.net/ "BMS OF FIGHTERS") and other similar competitions.
```

| :-: | :-- | :-- | :-- | :-- | :-- |
| [1](https://osu.ppy.sh/home/news/2020-11-30-a-labour-of-love) | [Kuba Oms - My Love](https://osu.ppy.sh/beatmaps/artists/13) | extremely valid (![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471), ![][flag_FR] [Shinwoir](https://osu.ppy.sh/users/8984574), ![][flag_DE] [Mao](https://osu.ppy.sh/users/2204515), ![][flag_FR] [Realazy](https://osu.ppy.sh/users/918297), and ![][flag_GB] [DeviousPanda](https://osu.ppy.sh/users/4966334)) | schoolboy fans (![][flag_RU] [Shadren](https://osu.ppy.sh/users/745494), ![][flag_UA] [wajinshu](https://osu.ppy.sh/users/6339790), ![][flag_RU] [Shmiklak](https://osu.ppy.sh/users/5504231), and ![][flag_RU] [Daycore](https://osu.ppy.sh/users/5596337)) | Team Red but Blue (![][flag_CA] [Redside](https://osu.ppy.sh/users/16039046), ![][flag_US] [Tofumang](https://osu.ppy.sh/users/4817223), ![][flag_CA] [Chompy](https://osu.ppy.sh/users/7427035), ![][flag_US] [Noffy](https://osu.ppy.sh/users/1541323), and ![][flag_CA] [Zer0-G](https://osu.ppy.sh/users/12577911)) | [Results](https://osu.ppy.sh/home/news/2021-04-27-results-a-labour-of-love) |

#### Best of/Community Choice

> The Best of/Community Choice is a voting contest that is being held annually to determine the most popular beatmaps <!-- https://osu.ppy.sh/home/news/2021-02-05-community-choice-2020-results --> in the eyes of the osu! community at each given year.

Reviewer 3: ```suggestion
The *Best of/Community Choice* contest is a voting contest that is held annually to determine the most popular beatmaps in the eyes of the osu! community each year. <!-- https://osu.ppy.sh/home/news/2021-02-05-community-choice-2020-results --> 
```

Author: think i'll keep this one as that comment is there as an indicator that the contest is indeed meant to sort out "the most popular" (as mentioned in the linked article) instead of "the best"

> At its launch, the contest was named "Best of" as the initial premise and expectation of the contest was to encourage members of the community to vote for what they consider as "the best beatmaps of the year". However, as the game's playerbase continue to grew up in number over time, the contest was inevitably shifting more and more into a popularity contest each year which prompted the contest to be relaunched as "Community Choice" in 2017.

Reviewer 3: ```suggestion
At its launch, the contest was named "Best of" as the initial premise and expectation of the contest was to encourage members of the community to vote for what they consider as "the best beatmaps of the year"; however, as the game's playerbase continue to grow in number over time, the contest was inevitably shifting more and more into a popularity contest each year which prompted the contest to be relaunched as "Community Choice" in 2017.
```

| [2020](https://osu.ppy.sh/home/news/2021-01-21-community-choice-2020-voting-open) | ![osu!][osu!]/![osu!taiko][osu!taiko]/![osu!catch][osu!catch]/![osu!mania][osu!mania] | [DADDY ! DADDY ! DO ! feat. Suzuki Airi (TV Size)](https://osu.ppy.sh/beatmapsets/1147361) hosted by ![][flag_GB] [hypercyte](https://osu.ppy.sh/users/9155377) | [Calliope Mori - Excuse My Rudeness, but Could You Please RIP?<3 (t+pazolite Remix)](https://osu.ppy.sh/beatmapsets/1278695) hosted by ![][flag_DE] [Capu](https://osu.ppy.sh/users/2474015) | [Camellia feat. Nanahira - finorza](https://osu.ppy.sh/beatmapsets/1151836) hosted by ![][flag_KR] [Rocma](https://osu.ppy.sh/users/566276) | [Camellia - We Could Get More Machinegun Psystyle! (And More Genre Switches)](https://osu.ppy.sh/beatmapsets/1034114) hosted by ![][flag_GB] [Hydria](https://osu.ppy.sh/users/808176) | [Results](https://osu.ppy.sh/home/news/2021-02-05-community-choice-2020-results) |

#### Fan Art Contest

> The Fan Art Contest is a drawing contest where participating artists from all across the community are being challenged to create the best fan art for osu! involving [osu!'s mascots](wiki/Mascots) across varying themes and art concepts.

Reviewer 3: ```suggestion
The *Fan Art Contest* is a drawing contest where participating artists from all across the community are challenged to create the best fan art for osu! involving [osu!'s mascots](wiki/Mascots) across various themes and art concepts.
```

| [#23 (Halloween 2020)](https://osu.ppy.sh/home/news/2020-09-18-halloween-2020-fanart-contest) | Halloween | ![][flag_BR] [CupcakeCrazier](https://osu.ppy.sh/users/12063010) | ![][flag_PH] [RianneRyan](https://osu.ppy.sh/users/17165886) | ![][flag_NZ] [SJPumpkin](https://osu.ppy.sh/users/4840900) | [Results](https://osu.ppy.sh/home/news/2020-10-11-halloween-fanart-contest-results) |
| [#24 (Winter 2020)](https://osu.ppy.sh/home/news/2020-09-11-winter-sports-2020-fanart-contest) | Winter Sports | ![][flag_US] [Tofumang](https://osu.ppy.sh/users/4817223) | ![][flag_ID] [CheekyIfa](https://osu.ppy.sh/users/14406940) | ![][flag_RU] [Dem4eg-](https://osu.ppy.sh/users/7206818) | [Results](https://osu.ppy.sh/home/news/2020-12-02-winter-2020-arrives) |
| [#25 (Spring 2021)](https://osu.ppy.sh/home/news/2021-04-23-spring-2021-fanart-contest) | A Day In The Park | ![][flag_US] [Tofumang](https://osu.ppy.sh/users/4817223) | ![][flag_MY] [Kyvrin](https://osu.ppy.sh/users/11589256) | ![][flag_BG] [risuuna](https://osu.ppy.sh/users/7266506) | [Results](https://osu.ppy.sh/home/news/2021-05-20-spring-fanart-contest-results) |

#### osu! Beatmap Blueprints Contest

> The osu! Beatmap Blueprints Contest is a one-off contest held in commemoration of the launch of [Beatmap Blueprints](https://osu.ppy.sh/store/products/9) in the [osu!store](https://osu.ppy.sh/store/listing). In this contest, participating users are competing to create the best blueprint of a beatmap in any visual medium using all the assets provided in the Beatmap Blueprints merch pack.

Reviewer 3: ```suggestion
The *osu! Beatmap Blueprints Contest* is a one-off contest held in commemoration of the launch of [Beatmap Blueprints](https://osu.ppy.sh/store/products/9) in the [osu!store](https://osu.ppy.sh/store/listing). In this contest, participating users competed to create the best blueprint of a beatmap in any visual medium using all the assets provided in the Beatmap Blueprints merchandise pack.
```

| Contest No. | ![Gold Crown][GCrown] | ![Silver Crown][SCrown] | ![Bronze Crown][BCrown] | Results |
| :-: | :-- | :-- | :-- | :-- |
| [#1](https://osu.ppy.sh/home/news/2015-03-20-osu-beatmap-blueprints-available-contest) | *N/A* | *N/A* | *N/A* | *N/A* |

#### osu!idol

> osu!idol is a singing/song cover contest where participating singers from all across the community are singing their way out either as a solo or a duet act across multiple judging and community voting stages. The contest is named after the [Idols franchise](https://en.wikipedia.org/wiki/Idols_(franchise)) and is mainly inspired by various televised singing competition shows.

Reviewer 3: ```suggestion
*osu!idol* is a singing/song cover contest where participants from all across the community compete to create the best song cover (either solo or with a partner) across multiple judging and community voting stages. The contest is named after the [Idols franchise](https://en.wikipedia.org/wiki/Idols_(franchise) "Wikipedia") and is mainly inspired by various televised singing competition shows.
```

| [#5 (2018)](https://osu.ppy.sh/home/news/2018-07-01-osu-idol-2018-auditions-now-open) | ![][flag_US] [Will Stetson](https://osu.ppy.sh/users/4909088) | ![][flag_ID] [Zeva\[FL\]](https://osu.ppy.sh/users/2805170) | ![][flag_US] [MjIsMaster](https://osu.ppy.sh/users/9922896) | [Will Stetson - Snow Drive](https://assets.ppy.sh/contests/71/entries/Snow%20Drive.mp3) | [Results](https://osu.ppy.sh/home/news/2018-11-05-osu-idol-2018-finals-community-voting-results) |
| [#6 (2019)](https://osu.ppy.sh/home/news/2019-07-02-osu-idol-2019-auditions-now-open) | ![][flag_PH] [iamALieee](https://osu.ppy.sh/users/10093980) | ![][flag_US] [aceredshirt13](https://osu.ppy.sh/users/13736238) | ![][flag_US] [moof](https://osu.ppy.sh/users/8457202) | [iamALieee - White First Love](https://assets.ppy.sh/contests/83/entries/osu!idol%202019%20-%20iamALieee%20-%20White%20First%20Love.mp3) | [Results](https://osu.ppy.sh/home/news/2019-12-21-osu-idol-2019-finals-community-voting-results) |
| [#7 (2020)](https://osu.ppy.sh/home/news/2020-09-01-osu-idol-2020-auditions-now-open) | *N/A* | *N/A* | *N/A* | *N/A* | *N/A* |

#### osu!remix

> osu!remix is a song remix contest where participating musicians from all across the community are being tasked to the best remix of a song that has been widely known throughout the osu! community in one way or another. It is the first ever contest to be handled via the [Community Contests listing](https://osu.ppy.sh/community/contests) page.

Reviewer 3: ```suggestion
*osu!remix* is a song remix contest where participating musicians from all across the community are tasked to create the best remix of a chosen song that has been widely known throughout the osu! community in one way or another. It is the first ever contest to be handled via the [Community Contests listing](https://osu.ppy.sh/community/contests) page.
```

| Contest No. | Remixed Song | ![Gold Crown][GCrown] | ![Silver Crown][SCrown] | ![Bronze Crown][BCrown] | Results |
| :-: | :-- | :-- | :-- | :-- | :-- |
| [#1](https://osu.ppy.sh/home/news/2016-05-06-osu-circles-remix-contest) | [nekodex - circles!](https://osu.ppy.sh/beatmaps/artists/1) | ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471) | ![][flag_FR] [Kurokotei](https://osu.ppy.sh/users/398275) | ![][flag_NL] [happy30](https://osu.ppy.sh/users/27767) | [Results](https://osu.ppy.sh/home/news/2016-08-29-osuremix-contest-1-results) |

#### Skinning Contest

> The Skinning Contest is a skinning contest open to skin creators from all across the community. 

Reviewer 3: this is very vague? may be worth putting a bit more detail in here

Reviewer 2: ```suggestion
The **Skinning Contest** is a skinning contest where artistic creators from all across the community compete to create the best skin in a limited time. 
```

Author: going with this yup

| Contest No. | ![Gold Crown][GCrown] | ![Silver Crown][SCrown] | ![Bronze Crown][BCrown] | Results |
| :-: | :-- | :-- | :-- | :-- |
| [#1](wiki/Contests/Skinning_Contest/) | [FREEDOM DiVE↓](https://osu.ppy.sh/community/forums/topics/1293207) by ![][flag_US] [BTMC](https://osu.ppy.sh/users/3171691) | [Minimalist 2.0](https://osu.ppy.sh/community/forums/topics/1286223) by ![][flag_EE] [Krizto](https://osu.ppy.sh/users/13474556) | [notMiku](https://osu.ppy.sh/community/forums/topics/1292954) by ![][flag_RU] [NikStrim](https://osu.ppy.sh/users/10577559) | [Results](https://osu.ppy.sh/home/news/2021-05-27-skinning-contest-results) |

## Unofficial/Community-Run Contests

> Unofficial contests are contests that are being hosted by members of the community. Contest organizers who are looking to add their contest to the list below are welcome to do so by opening a GitHub pull request in the [osu-wiki repository](https://github.com/ppy/osu-wiki) following the [osu! wiki contribution guide](/wiki/osu!_wiki_Contribution_Guide).

Reviewer 3: ```suggestion
*Unofficial contests* are contests that are hosted by members of the community. Contest organizers who are looking to add their contest to the list below are welcome to do so by opening a GitHub pull request in the [osu-wiki repository](https://github.com/ppy/osu-wiki "GitHub") following the [osu! wiki contribution guide](/wiki/osu!_wiki_Contribution_Guide).
```

### osu! Beatmapping World Championship <!-- Yes, OBWC is not an official contest according to https://twitter.com/paichiru/status/1408900730149249025 -->

> osu! Beatmapping World Championship is a team-based beatmapping contest where participating teams consisting of mappers from the same country, region, or even continent (to some extent) are battling their mapping skills out against each other across multiple songs and mapping stages. The contest is known for exclusively making use of songs available in the [Featured Artist](wiki/Featured_Artists) library during its runs.

Reviewer 3: ```suggestion
*osu! Beatmapping World Championship* is a team-based beatmapping contest where participating teams consisting of creators from the same country, region, or even continent (to some extent) battle their mapping skills out against each other across multiple songs and mapping stages. The contest is known for exclusively making use of songs available in the [Featured Artist](wiki/Featured_Artists) library during its runs.
```

| [#2 (2020)](/wiki/Contests/oBWC/2) | ![][flag_GB] United Kingdom ([hypercyte](https://osu.ppy.sh/users/9155377), [Aistre](https://osu.ppy.sh/users/4879380), [Altai](https://osu.ppy.sh/users/5745865), [Chugger](https://osu.ppy.sh/users/4491713), [DeviousPanda](https://osu.ppy.sh/users/4966334), [pocket-](https://osu.ppy.sh/users/6808091), and [Yusomi](https://osu.ppy.sh/users/4174940)) | ![][flag_AR] Argentina ([MaestroSplinter](https://osu.ppy.sh/users/6707918), [chucentry](https://osu.ppy.sh/users/2498731), [Lince Cosmico](https://osu.ppy.sh/users/6070370), and [Megafan](https://osu.ppy.sh/users/6632605)) | ![][flag_BR] Brazil ([Net0](https://osu.ppy.sh/users/5099768), [Enerugi](https://osu.ppy.sh/users/4475985), [Faito](https://osu.ppy.sh/users/9706291), [Kowari](https://osu.ppy.sh/users/5404892), [Maot](https://osu.ppy.sh/users/3914271), [Sakura Airi](https://osu.ppy.sh/users/8682057), and [Trynna](https://osu.ppy.sh/users/2652951)) | [Results](https://osu.ppy.sh/home/news/2020-10-22-beatmapping-world-championship-2020-concludes) |

### Pending Cup

> The Pending Cup is a beatmapping contest that is being held yearly by prominent members of the Chinese mapping community. Each year, the contest features up to three songs and/or categories that mappers from all across the community are free to compete and submit their entries in.

Reviewer 3: ```suggestion
The *Pending Cup* is a beatmapping contest that is held yearly by prominent members of the Chinese mapping community. Each year, the contest features up to three songs and/or categories that creators from all across the community are free to compete and submit their entries in.
```

| Vocal² | ![osu!][osu!] | ![][flag_KR] [Heilia](https://osu.ppy.sh/users/9823042) | ![][flag_DE] [jamesjan3](https://osu.ppy.sh/users/6260705) | ![][flag_CN] [Garden](https://osu.ppy.sh/users/2849992) | [Results](https://osu.ppy.sh/community/forums/topics/1198663) |
| Instrumental³ | ![osu!][osu!] | ![][flag_RU] [xbopost](https://osu.ppy.sh/users/6842421) | ![][flag_ID] [\[Keqing\]](https://osu.ppy.sh/users/8972308) | ![][flag_KR] [Acylica](https://osu.ppy.sh/users/1943309) | [Results](https://osu.ppy.sh/community/forums/topics/1198663) |

¹ The Featured Artist category provides mappers with a choice of two different songs to be mapped: Zekk - Re\_construct and Shuu no Hazama \[Rainshower\].

² The Vocal category provides mappers with a choice of two different songs to be mapped: nonoc - memento and DUSTCELL - DERO.

³ The Instrumental category provides mappers with a choice of two different songs to be mapped: onoken - Felys final remix and Nekomata Master - The Sky of Sadness.

### Newspaper Cup

> The Newspaper Cup is a beatmapping contest that is similar to Pending Cup in nature but with a much greater emphasis on novice mappers. As with Pending Cup, the contest is also being held yearly by prominent members of the Chinese mapping community and has since been developed from a contest that was particularly intended for Chinese-speaking members of the community to a global-scale contest.

Reviewer 3: ```suggestion
The *Newspaper Cup* is a beatmapping contest that is similar to the Pending Cup in nature, although with a much greater emphasis on novice creators. As with the Pending Cup, the contest is also held yearly by prominent members of the Chinese mapping community and has since been grown from a solely Chinese-speaking audience to a global an international one.
```
